types/mongoose/mongoose-tests.ts

```diff
@@ -1278,7 +1284,7 @@ MongoModel.find({}).$where('indexOf("val") !== -1').exec(function (err, docs) {
   docs[0].__v;
 });
 MongoModel.findById(999, function (err, doc) {
-  var handleSave = function(err: Error, product: mongoose.Document, numAffected: number) {};
+  var handleSave = function(err: Error, product: mongoose.Document) {};
```

Comment:
`numAffected` is still allowed http://mongoosejs.com/docs/api.html#document_Document-save

Reply:
Nevermind! https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23070#discussion_r163488419